plugins/aws_sts/app/decorators/stage_decorator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+Stage.class_eval do
+  validate :validate_arn_syntax, if: -> { self&.aws_sts_iam_role_arn.present? }
+  validates :aws_sts_iam_role_session_duration, allow_blank: true,
+                                                numericality: {
```

Comment:
so much wasted whitespace, 2-space indent plz :D 
something like:
```
validates(
  :aws_sts_iam_role_session_duration, 
  allow_blank: true,
  foo: bar
)
```

Reply:
I know, I wasn't going to argue with rubocop :P I'll see what it thinks when I use brackets